### PR TITLE
make <a> optional on section header for archive

### PR DESCRIPTION
**Pull request by border0464111**
maybe it would be good to rephrase the option description. (as im not native english speaker )

### CODE REVIEW

**Comment by border0464111** [comment]
Travis fail at php7. But my patch should is php7 conform. Does travis have been run aganst the whole master repository in his current state ?

**Comment by ajaydsouza** [comment]
I've updated the travis tests file so it shouldn't fail on 7.x

File: includes/admin/register-settings.php [hunk 1]
```diff
@@ -234,6 +234,13 @@ function wzkb_get_registered_settings() {
 					'type'             => 'text',
 					'options'          => 'kb-tags',
 				),
+				'clickable_section'     => array(
+					'id'               => 'clickable_section',
+					'name'             => esc_html__( 'Section title clickable', 'knowledgebase' ),
```
Comment (ajaydsouza) [inline includes/admin/register-settings.php]: Link section title

File: includes/admin/register-settings.php [hunk 2]
```diff
@@ -234,6 +234,13 @@ function wzkb_get_registered_settings() {
...
+					'desc'             => esc_html__( 'Allow Section title to be a link to his specific section page.', 'knowledgebase' ),
```
Comment (ajaydsouza) [inline includes/admin/register-settings.php]: If selected, the title of each section of the knowledgebase will be linked to its own page

File: includes/main.php [hunk 3]
```diff
@@ -227,11 +227,17 @@ function wzkb_list_posts_by_term( $term, $level ) {
  */
 function wzkb_article_header( $term, $level ) {
 
-	$output = '
- <h3 class="wzkb_section_name wzkb-section-name-level-' . $level . '">
-  <a href="' . get_term_link( $term ) . '" title="' . $term->name . '" >' . $term->name . '</a>
- </h3>
- ';
+	$wzkb_options = get_option('wzkb_settings',true);
```
Comment (ajaydsouza) [inline includes/main.php]: WordPress CS requires a space before and after the bracket.
Use `wzkb_get_option` to get the option we need.
e.g. `$clickable_section = wzkb_get_option( 'clickable_section', true );`

File: includes/main.php [hunk 4]
```diff
@@ -227,11 +227,17 @@ function wzkb_list_posts_by_term( $term, $level ) {
...
+
+	$output = '<h3 class="wzkb_section_name wzkb-section-name-level-' . $level . '">';
+
+	if ( $wzkb_options['clickable_section'] ) {
```
Comment (ajaydsouza) [inline includes/main.php]: See line 230

**Comment by ajaydsouza** [comment]
Hi, I've requested a few changes to make it similar to the rest of the plugin code

File: includes/admin/register-settings.php (hunk 1 above)
Comment (border0464111) [inline includes/admin/register-settings.php]: > Link section title 
As the content of the name field ?

Comment (ajaydsouza) [inline includes/admin/register-settings.php]: Sorry, should have been clearer. Please call the name as "Link section title"

File: includes/admin/register-settings.php (hunk 2 above)
Comment (ajaydsouza) [inline includes/admin/register-settings.php]: This is for 'desc'

**Comment by border0464111** [comment]
updated.

**Pull request merged**